### PR TITLE
:sparkles: (UI): UI Kit Cards Component

### PR DESCRIPTION
**Pull request by MatthewRobinMarsden**
The angular implementation of the cards component. This component is part of the general UI Kit library created by the UI Kit team. Architecture team @COS-301/visual-basic.

### CODE REVIEW

**Comment by JurassicRegz** [comment]
Everything is good, fix should release tomorrow for making sure that the same meta test runs when changing labels inside the PR.

**Comment by evilcomrade** [comment]
> Everything is good, fix should release tomorrow for making sure that the same meta test runs when changing labels inside the PR.

Awesome thanks for the speedy assistance :D

File: libs/client/shared/components/cards/ui/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Cards
+
+This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 13.3.1.
```
Comment (arneschreuder) [inline libs/client/shared/components/cards/ui/README.md:3]: No, please generate libraries using `nx` not using `ng`. Use the VSCode NX console extension and use `nrwl/angular library`

**Review by arneschreuder — CHANGES_REQUESTED** [review]
Please generate the libs using the correct tooling. This is what I was talking about in class today :)

Also, I suggest making this PR again from your feature branch and not from your own develop. So git checkout -b feature/my-branch and then make the PR from there.

**Pull request closed without merging**